src/client/Client.js

```diff
@@ -414,6 +415,17 @@ class Client extends BaseClient {
       query.set('guild_id', guildID);
     }
 
+    if (options.additionalScopes) {
+      const scopes = options.additionalScopes;
+      if (!Array.isArray(scopes)) {
+        throw new TypeError('INVALID_TYPE', 'additionalScopes', 'Array of Invite Scopes', true);
+      } else if (scopes.some(scope => !InviteScopes.includes(scope))) {
+        const invalidScope = scopes.find(scope => !InviteScopes.includes(scope));
+        throw new TypeError('INVALID_ELEMENT', 'Array', 'additionalScopes', invalidScope);
+      }
```

Comment:
You can avoid a second iteration, since the previous `if` always throws when executed, the `else` branch is useless, which allows us to make a new `if`, and thus, we can do the following to avoid a second iteration:
```suggestion
      }
      const invalidScope = scopes.find(scope => !InviteScopes.includes(scope));
      if (invalidScope) {
        throw new TypeError('INVALID_ELEMENT', 'Array', 'additionalScopes', invalidScope);
      }
```

Reply:
I literally thought about this, then didn't do it 🤦,  thanks!